src/services/util/util.service.ts

```diff
@@ -46,4 +54,33 @@ export class UtilService {
       new ContributionInputData(roomId, users),
     );
   }
+
+  public save(roomId: string, content: DeepReadonly<ExcalidrawContent>) {
+    return this.integrationService.save(
+      new SaveInputData(roomId, JSON.stringify(content)),
+    );
+  }
```

Comment:
this method cannot throw, it will return a class with error info (see server.ts:584). Then the error can be handled by whichever code is using the method - is there a better way?

Reply:
`WhiteboardIntegrationAdapterService.save` is async and handles exceptions, but returns a class with an error message instead. How is that?